integration/template/azure_resource_charts_values.go

```diff
@@ -0,0 +1,16 @@
+package template
+
+var AzureResourceChartValues = `clusterName: ${CLUSTER_NAME}
+commonDomain: ${COMMON_DOMAIN_GUEST_NO_K8S}
+commonDomainResourceGroup: ${COMMON_DOMAIN_RESOURCE_GROUP}
+sshUser: "test-user"
+sshPublicKey: ${IDRSA_PUB}
+azure:
+  location: ${AZURE_LOCATION}
+  vmSizeMaster: "Standard_D2s_v3"
+  vmSizeWorker: "Standard_D2s_v3"
```

Comment:
may be think of smaller vm size?

Reply:
e.g. there is smallest (two times smaller than this one) VM in prev. generation that supports managed disks `Standard_DS1_v2`

Reply:
good point, will try this

Reply:
tried it and ran into timeout (30mn). see https://circleci.com/gh/giantswarm/azure-operator/933
not 100% sure though that was due to vm being too small